### ISSUE TITLE
Make changes in Settings page to adapt to change in Settings API

### ISSUE THREAD

**Issue by akshatnitd** [fossasia/susi_android#2235]
**Describe the chore**
- Make changes in Settings page to adapt to change in Settings API.
Link to the API - https://github.com/fossasia/susi_server/blob/development/src/ai/susi/server/api/aaa/ChangeUserSettings.java

**Would you like to work on the issue?**
No

**Comment by jogendra** [comment]
Refer to changes in SUSI iOS here: https://github.com/fossasia/susi_iOS/issues/563

**Comment by atm1504** [comment]
@akshatnitd I found that few parameters were added in the possibleKeys. I want to bring to your notice that we currently don't have those features available in our app. 
@iamareebjamal @batbrain7 @arundhati24 please confirm it.

**Comment by arundhati24** [comment]
@atm1504 Yes, there are many keys in the possibleKeys array. You need to consider only those keys which are applicable in the existing app.
